src/GeoJson/Geometry/Polygon.php

```diff
@@ -12,12 +12,10 @@
  */
 class Polygon extends Geometry
 {
-    protected $type = 'Polygon';
+    protected string $type = 'Polygon';
 
     /**
-     * Constructor.
-     *
-     * @param float[][][]|LinearRing[] $linearRings
+     * @param array<LinearRing|array<Point|array<int|float>>> $linearRings
```

Comment:
I was going to ask where `Point` came from but realized this is accepted by the `LinearRing` constructor. :+1:

Reply:
I remember that my head started spinning as well when I added all those type definitions 😂